java/.ci/Jenkinsfile

```diff
@@ -769,7 +769,29 @@ pipeline {
                     }
                 }
             }
-            
+            stage('Parallel Execution 9'){
+                parallel{
+                    stage('PubSubLite To GCS') {
+                        steps{
+                            sh '''
+                                export GCP_PROJECT=yadavaja-sandbox
+                                export REGION=us-west1
+                                export GCS_STAGING_LOCATION="gs://python-dataproc-templates-temp"
+
+                                cd java
+
+                                bin/start.sh \
+                                -- --template PUBSUBLITETOGCS \
+                                --templateProperty pubsublite.to.gcs.input.subscription.url=projects/$GCP_PROJECT/locations/$REGION/subscriptions/psltobt-sub \
```

Comment:
Will $GCP_PROJECT work? Or is the $GCP_PROJECT_NUMBER is required?
@balajiss2

Reply:
The $GCP_PROJECT_NUMBER is required. The $GCP_PROJECT did not work. I tried that already

Reply:
It is working fine though with $GCP_PROJECT. Why do we need to change it, in python template also $GCP_PROJECT was used